astro/src/content/quickstarts/quickstart-php-laravel-web.mdx

```diff
@@ -0,0 +1,349 @@
+
+### Run FusionAuth And {frontmatter.technology} Via Docker
+
+You'll find a Docker Compose file `docker-compose.yml` and an environment variables configuration file `.env` in the root folder of the repository.
+
+Assuming you have Docker installed, you can start FusionAuth on your machine with the following.
+
+```shell
+docker compose up -d
+```
+
+This will start five containers, one each for FusionAuth, Postgres, Elasticsearch, MariaDB, and {frontmatter.technology}. Wait for all five containers to start.
+
+Here you are using a bootstrapping feature of FusionAuth, called [Kickstart](/docs/v1/tech/installation-guide/kickstart). When FusionAuth starts for the first time, it will look at the `kickstart/kickstart.json` file and configure FusionAuth to your specified state.
+
+<Aside type="note">
+If you ever want to reset the FusionAuth system, delete the volumes created by `docker-compose` by executing `docker compose down -v`, then rerun `docker compose up -d`.
+</Aside>
+
+FusionAuth will be configured with these settings:
+
+* Your client Id is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is `super-secret-secret-that-should-be-regenerated-for-production`.
+* Your example username is `richard@example.com` and the password is `password`.
+* Your admin username is `admin@example.com` and the password is `password`.
+* The base URL of FusionAuth is `http://localhost:9011/`.
+
+You can log in to the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want to, but with Docker and Kickstart, everything will already be configured correctly.
+
+<Aside type="caution">
+  The `.env` and `kickstart.json` files contain passwords. In a real application, always add these files to your `.gitignore` file and never commit secrets to version control.
+</Aside>
+
+### The Basic {frontmatter.technology} Application
+
+While this guide builds a new {frontmatter.technology} project, you can use the same method to integrate your existing project with FusionAuth.
+
+The {frontmatter.technology} [documentation](https://laravel.com/docs/10.x#your-first-laravel-project) suggests many ways of creating a new project. This guide uses the standard [Bitnami Laravel Docker image](https://hub.docker.com/r/bitnami/laravel), which creates the project for you. This image also contains a version of PHP, so you don't need to install PHP on your physical machine.  If you browse to [http://localhost:5001](http://localhost:5001), you'll see the site is running.
+
+<Aside type="note">
+  It may take a few more minutes for the host to come up on http://localhost:5001 after you run the `docker compose up -d` command as the Laravel container updates dependencies and executes migrations.
+</Aside>
+
+All the starter app files are available for editing in the `your-application` folder and mapped to the {frontmatter.technology} Docker volume. Composer packages are already in the `vendor` folder, so you don't need to download them.
```

Comment:
I cloned the repo and didn't see this directory. Do I need to create it? What am I missing?

Reply:
The directory would be created once the Laravel container starts up successfully but I have changed the implementation not to use docker in this PR https://github.com/FusionAuth/fusionauth-quickstart-php-laravel-web/pull/2